cfme/infrastructure/pxe.py

```diff
@@ -704,6 +691,39 @@ def delete(self, cancel=True):
             navigate_to(self, 'Details')
 
 
+@attr.s
+class SystemImageTypeCollection(BaseCollection):
+    """ Collection class for SystemImageType. """
+
+    ENTITY = SystemImageType
+
+    def create(self, name, provision_type, cancel=False):
+        """
+        Creates a System Image Type object
+
+        Args:
+            name: It contains name of the System Image Type
+            provision_type: Type on Image. i.e Vm and Instance or Host
+            cancel (boolean): Whether to cancel out of the creation.  The cancel is done
+                after all the information present in the SIT has been filled in the UI.
+        """
+
+        system_image_type = self.instantiate(name, provision_type)
+
+        view = navigate_to(self, 'Add')
+        view.fill({'name': name, 'type': provision_type})
+        main_view = self.create_view(PXESystemImageTypesView)
```

Comment:
**REQUIRED:** To assist in the flow of reading, ```main_view``` definition should be moved below the conditional